### DISCUSSION TITLE
HTTP timeline after/before arguments not working

### DISCUSSION THREAD

**Discussion by maxdd** [blakeblackshear/frigate#10938] [General]
Hello,
i was playing around with HTTP API and i noticed a strange behaviour which i cannot understand
When i type

`http://192.168.1.52:5000/api/timeline?after=1712666897`

the result is

![image](https://github.com/blakeblackshear/frigate/assets/6155973/2bab4f3b-4d00-499c-a4e9-94fb7f338f98)

As you can see the returned queries are all before the requested time e.g. 1712655397.073745 
Note that the result of they query is capped to 100

**Comment by NickM-27** [comment]
That's because `after` is not a valid query arg for the timeline api https://docs.frigate.video/integrations/api#get-apitimeline
